ocrd_network/ocrd_network/process_helpers.py

```diff
@@ -0,0 +1,47 @@
+import json
+from typing import List
+
+from ocrd import Resolver
+from ocrd.processor.helpers import run_cli, run_processor
+
+
+# A wrapper for run_processor() and run_cli()
+def run_single_execution(
```

Comment:
I would stick to `run_single_execution` for the current time being since `process` is too wide and does not hint at what it refers to directly. Also, there are `process_message` and `processor_job_task`. As agreed before, the naming conventions should be discussed at the later stages when everything is working and ready for merge/release.

Reply:
> In essence, what this does is delegate to ocrd.Processor.process. So why not just call it ocrd_network.process_helpers.process (or even better ocrd.processor.process)?

Because I wanted a clear differentiation between `ocrd.processor.process` and `ocrd_network.process_helpers.process`. 

> After all, process is what a processor should do.

Yes, but we have Processing Worker and Processor Server context, not Processor. 

> I agree there's a clash with `process_message` (as imperative of OcrdProcessingMessage), but I would rather change the latter (`consume_message` would seem a better fit) than block the former interpretation. 

There is already `on_consumed_message` which decodes `bytes` to `OcrdProcessingMessage`, handles RabbitMQ stuff, and passes the `OcrdProcessingMessage` to `process_message`. The `process_message` then, processes the message by resolving data inside the message.

> To me, what also speaks against `run_single_execution` is that `run` is redundant with `execute`.

Agree, my bad. `invoke_execution`/`invoke_processing` or something along these lines would have been better. Since I don't see this potentially conflicting anywhere in the future, I have refactored `run_single_execution` to `process`.

> IMHO `ocrd_network.ProcessorServer.processor_job_task` should be an imperative verb, too. And it's also redundant btw (`job` vs. `task`). How about `consume_processor_job` or `do_processor_job`, or (replacing the term processor job througout) `create_task` + `run_task`?

Agree. In my mind, conceptually, the job was referring to the job model and I made the mistake to put `job` and `task` together, which is bad. I don't think `consume` fits here, nothing is being consumed. 

In conclusion, I have refactored: 
`run_single_execution` -> `invoke_processor`
`create_processor_job_task` -> `create_processor_task` 
`processor_job_task` -> `run_processor_task`

Thanks for your input, this looks better now!